scenarios/14e_Saorduc_Swamps.cfg

```diff
@@ -29,7 +29,13 @@
         carryover_percentage=100
         linger_mode=no
     [/endlevel]
-    {END_IF_WITHOUT_ELSE}
+    # TODO: maybe give the option to let the saurians join you?
+    {ELSE}
+    # TODO: test display of these messages:
+    {MESSAGE ({LIZ}) () ({LIZ}) _"We are more than jusssst training fodder for your army!"}
+    {MSGW_Gawen _"I'm sorry, but that's just the way things go."}
+    {MSGW_Lorin _"Don't apologize to them, Gawen! They are filthy lizards who deserve to die!"}
```

Comment:
This line is a bit better, but she'd probably say something a bit less excitedly and more matter of fact, like "Don't apologize to these filthy lizards." and leave it at that.

Reply:
ok changed in 61ba872